src/Interface/TicketInterface.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+    THIS IS NOT A UI COMPONENT. This interface defines our data structure 
+    for a single ticket. This is NOT a UI component but rather a declaration
+    of the properties that exist on a Ticket instance.
+
+    IMPORTANT: If you want to have a variable of type Ticket anywhere else
+    in the program you MUST import the interface for it from this file!
+    Trying to create an object with the same properties is NOT sufficient
+    so the variable must be declared with a type of Ticket or TypeScript
+    will get mad!
+*/
+
+export interface Ticket {
```

Comment:
This has been moved into TicketItem please delete

Reply:
The point is that I don't think it should be in ticketItem. I've moved it out so it can function as its own interface and updated everything accordingly.